qutebrowser/utils/log.py

```diff
@@ -291,6 +303,20 @@ def _init_formatters(level, color, force_color, json_logging):
     return console_formatter, ram_formatter, html_formatter, use_colorama
 
 
+def change_console_formatter(level):
+    """Change console formatter based on level.
+
+    Args:
+        level: The numeric logging level
+    """
+    if isinstance(console_handler.formatter, ColoredFormatter):
+        use_colors = console_handler.formatter.use_colors()
+        console_fmt = get_format(level)
+        console_format = ColoredFormatter(console_fmt, DATEFMT, '{',
```

Comment:
I think for consistency I should leave it as `console_fmt`, but I will change to `console_formatter`.  Unless you want to change every instance of `console_fmt` to `console_format`.